347. Top K Frequent Elements/TopKFrequentElementsStep3.java

```diff
@@ -0,0 +1,32 @@
+public class TopKFrequentElementsStep3 {
+    /*
+     * 11m15s
+     * 時間計算量:O(nlogk)
+     * 空間計算量:O(n)
+     */
+    class Solution {
+        record NumAndCount(int num, int count) {}
+
+        public int[] topKFrequent(int[] nums, int k) {
+            Map<Integer, Integer> numToCount = new HashMap<>();
+            for (int num : nums) {
+                int lastCount = numToCount.getOrDefault(num, 0);
+                numToCount.put(num, lastCount + 1);

```

Comment:
加算前の count なのは明示しなくてもわかる気がしたので、 `lastCount` を `count` にしてしまってもいいかなと思いました

Reply:
この使い方に慣れてしまったので、変数化すらせず一行で書かれていても違和感ないなって思いました
```suggestion
                numToCount.put(num, numToCount.getOrDefault(num, 0) + 1);
```

Reply:
ありがとうございます。
いただいたソースを読んでみると、思っていたより分かりやすかったです。
自分で書くと間違えそうなのですが、慣れの問題かもしれません。